index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <div id="main">
```

Comment:
instead of using div pleas use semantic html tag in this case use _<header></header>_ tag.

Reply:
- [ ] Please when you give a review be more clear and give an explanation, a "why" you suggest the change to allow the student to understand very well 🙃 😉, also use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.